src/attribute_type.rs

```diff
@@ -295,6 +308,15 @@ impl ToString for SdpAttributeRemoteCandidate {
     }
 }
 
+impl AnonymizingClone for SdpAttributeRemoteCandidate {
+    fn masked_clone(&self, anon: &mut StatefulSdpAnonymizer) -> Self {
+        let mut masked = self.clone();
```

Comment:
I think the only value you need to clone without masking it is component in here. Thus I would suggest to invoke the constructor instead.